src/label-for-moment/label-for-moment.test.js

```diff
@@ -0,0 +1,18 @@
+import labelForMoment from './label-for-moment';
+import moment from 'moment';
+import _ from 'lodash';
+
+describe('_.labelForMoment()', () => {
+  it('should return in 00:00am format if the moment is in today', () => {
+    expect(labelForMoment(moment().subtract(20, 'second'))).to.equal('now');
```

Comment:
this isn't actually testing what 'it' says

not sure why this would be outputting now either - the code doesn't seem like it should be outputting that